src/index.js

```diff
@@ -1,6 +1,86 @@
 import './css/base.scss';
 
 import domUpdates from './domUpdates';
+import apiFetch from './fetch';
 
 import Trip from './Trip';
 import User from './User';
+
+const signOutButton = document.querySelector('.log-out');
+const costButton = document.querySelector('.input__cost-button');
+const confirmTripButton = document.querySelector('.input__confirm-button');
+
+let currentUser, trips, destinations, currentTrip;
+
+const sortUserTrips = (trips) => {
+  const currentDate = Date.now();
+  trips.forEach(trip => {
+    const tripDate = new Date(trip.date);
+    if (tripDate > currentDate || trip.status === 'pending') {
+      domUpdates.hidePendingHeader();
+      domUpdates.addPendingTrip(trip, destinations)
+    } else {
+      domUpdates.hidePastHeader();
+      domUpdates.addPastTrip(trip, destinations);
+    }
+  })
+}
+
+const onStartup = () => {
+  const usersPromise = apiFetch.getData('http://localhost:3001/api/v1/travelers');
+  const tripsPromise = apiFetch.getData('http://localhost:3001/api/v1/trips');
+  const destinationsPromise = apiFetch.getData('http://localhost:3001/api/v1/destinations');
+
+  Promise.all([usersPromise, tripsPromise, destinationsPromise])
+    .then(promises => {
+      currentUser = new User(promises[0].travelers[13]);
+      trips = promises[1].trips;
+      destinations = promises[2].destinations;
+      domUpdates.greetUser(currentUser);
+      domUpdates.populateDestinations(destinations);
+      sortUserTrips(currentUser.findUserTrips(trips));
+    })
+    domUpdates.setStartDate();
+}
+
```

Comment:
This file's organization is on FLEEK! Nice job on keeping the code clean concise. A small piece of feedback I have in your promise.all function, instead of assigning them to scoped variables inside the function, you can just use the method calls. It's going to the same thing, but will help with readability.